lib_xud/src/core/XUD_IoLoop.S

```diff
@@ -159,13 +159,6 @@ BadCrcAddr:
     //bt          r11, Pid_Sof_NoChan                    // TODO we should really CRC SOFs
     ldw         r11, sp[STACK_RXA_PORT]
 
```

Comment:
if we jump here now we run though a bunch of nops due to the align?

Reply:
grepping for BadCrcAddr doesn't give anything, so we can remove it

Reply:
Not sure what your nop-related note means, 
I guess we expect nops before anything which starts with .align, in this case, it looks like that
![image](https://github.com/user-attachments/assets/397af481-347c-48e6-85f2-b117bc7c7211)

Reply:
yes, nops are used to align where needed. Looks like moot though since the BadCrcAddr label is dead code